src/script/script.js

```diff
@@ -535,7 +535,23 @@ pc.extend(pc, function () {
                     this._enabledOld = this.enabled;
                     this.fire(this.enabled ? 'enable' : 'disable');
                     this.fire('state', this.enabled);
+
+                    if (! this._initialized && this.enabled) {
```

Comment:
@vkalpias 
I think that it is better to check `this.enable` only once.

```js
if (!this.enabled) return;
```
or
```js
if (this.enable) {
  if (! this._initialized) {
    ...
  }
  if (this.app._initialized && ! this._postInitialized) {
    ...
  }
}
```

Reply:
Yeah sure :)